### ISSUE TITLE
Hey! I try to install av-caster in CentOS7. Some error comes.

### ISSUE THREAD

**Issue by oddukgi** [bill-auger/av-caster#50]
I try to install av-caster.
[root@localhost ~]# yum install av-caster
Error: Package: av-caster-0.15-28.1.x86_64 (home_bill-auger)
           Requires: gstreamer1-plugins-ugly
 You could try using --skip-broken to work around the problem
*\* Found 1 pre-existing rpmdb problem(s), 'yum check' output follows:
pcp-compat-3.10.6-2.el7.x86_64 has missing requires of pcp-pmda-mysql

and then I try to install gstreamer1- plugins-ugly. I can't.
Is there way to install centOS7?

Error: Package: gstreamer-plugins-ugly-0.10.16-2.el6.rf.x86_64 (rpmforge)
           Requires: libcdio.so.10(CDIO_10)(64bit)
Error: Package: gstreamer-plugins-ugly-0.10.16-2.el6.rf.x86_64 (rpmforge)
           Requires: libcdio.so.10()(64bit)
 You could try using --skip-broken to work around the problem
*\* Found 1 pre-existing rpmdb problem(s), 'yum check' output follows:
pcp-compat-3.10.6-2.el7.x86_64 has missing requires of pcp-pmda-mysql

**Comment by bill-auger** [comment]
that centos package has not been updated in about 6 months for the same reason of broken packages and no one has actually asked about centos until now so it was not a priority to get it building again

on fedora the gstreamer packages are in the 'rpmfusion' third-party repo - the instructions are in the [README.md](https://github.com/bill-auger/av-caster/blob/master/README.md) under the fedora "build dependencies" - (put 23 for FEDORA_VERSION) - if that works for you then you may as well try installing the av-caster package for fedora because it is still building correctly and will be kept up to date

note that there are two different versions of gstreamer 0.10 and 1.x - it seems your package manager was trying to install 0.10 even though you asked for 1.x - this is incorrect behaviour - 1.x is not a replacement for 0.10 - they are separate version that may both be installed at the same time - av-caster requires version 1.6 or greater - once installed you can check with this command:

```
gst-launch-1.0 --version
```

plz let me know how this works out for you

**Comment by oddukgi** [comment]
Ok.. this is my gstreamer version.. 
[root@localhost ~]# gst-launch-1.0 --version
gst-launch-1.0 version 1.4.5
GStreamer 1.4.5

**Comment by bill-auger** [comment]
av-caster requires gStreamer version 1.6 or greater - if you can get gstreamer 1.6 or greater installed you should be able to build yourself even if the packages will not install

gStreamer has a chat room on freenode - you may be able to get help there installing  version 1.6

sry i dont have the time to build packages for all of the distros that i could - for the time being i am packaging for the following distros:
- Arch
- Debian 8, Ubuntu 15.10
- Fedora 23
- OpenSuse Tumbleweed , Suse SLE 12

**Comment by bill-auger** [comment]
also - if you need some help i am usually idling in the gitter chat https://gitter.im/bill-auger/av-caster
